tnt/v2/tnt_v2.2.0.yaml

```diff
@@ -0,0 +1,1970 @@
+          example: ADT
+          maxLength: 6
+        facilityCodeListProvider:
+          type: string
+          x-stoplight:
+            id: ikhscm43amwhh
+          description: The provider used for identifying the facility Code.
+          example: SMDG
+          enum:
+            - SMDG
+            - BIC
+        address:
+          $ref: '#/components/schemas/Address'
+    Address:
+      title: Address
+      x-stoplight:
+        id: 91zstl2wk79sh
+      type: object
+      description: "\t\nAddress related information."
+      properties:
+        name:
+          type: string
+          x-stoplight:
+            id: t2o6wpwquuk65
+          description: Name of the address.
+          example: Henrik
+          maxLength: 100
+        street:
+          type: string
+          x-stoplight:
+            id: 0z4vq2689z7k3
+          description: The name of the street of the party’s address.
+          example: Kronprincessegade
+          maxLength: 100
+        streetNumber:
+          type: string
+          x-stoplight:
+            id: jc4bsqwlm2i1r
+          description: The number of the street of the party’s address.
+          example: '54'
+          maxLength: 50
+        floor:
+          type: string
+          x-stoplight:
+            id: 5so5cxd9f70dg
+          description: The floor of the party’s street number.
+          example: 5. sal
+          maxLength: 50
+        postCode:
+          type: string
+          x-stoplight:
+            id: 5a9o88im9m6q5
+          description: The post code of the party’s address.
+          example: '1306'
+          maxLength: 10
+        city:
+          type: string
+          x-stoplight:
+            id: gndqjje7x94y1
+          description: The city name of the party’s address.
+          example: København
+          maxLength: 65
+        stateRegion:
+          type: string
+          x-stoplight:
+            id: zrciidsi6a2av
+          description: The state/region of the party’s address.
+          maxLength: 65
+          example: N/A
+        country:
+          type: string
+          x-stoplight:
+            id: gxqzzrbog4gtu
+          description: The country of the party’s address.
+          example: Denmark
+          maxLength: 75
+    DocumentReference:
+      title: DocumentReference
+      x-stoplight:
+        id: a2eh8cggmz2i5
+      type: object
+      description: An optional list of key-value (documentReferenceType-documentReferenceValue) pairs representing links to objects relevant to the event. The documentReferenceType-field is used to describe where the documentReferenceValue-field is pointing to.
+      properties:
+        documentReferenceType:
+          type: string
+          x-stoplight:
+            id: spt02mtugqsdv
+          description: "\r\nDescribes where the documentReferenceValue is pointing to."
+          example: BKG (Booking)
```

Comment:
This is not a 1:1 copy - here you have fixed the error in the published T&T spec. I suggest that in an upcoming version (must be a major release) it is fixed the other way around so we don't have the `(Booking)` as part of the enum value...
This is not to be fixed - just a comment